zio/zngio/index.go

```diff
@@ -21,6 +22,7 @@ const (
 )
 
 type TimeIndex struct {
+	lock       sync.Mutex
```

Comment:
Nit: Our traditional name for a sync.Mutex or sync.RWMutex is mu.

Reply:
🐄

Reply:
Took me a second to get this, but nice.

Reply:
I mean, we could go all in and use μ, which is a valid name in Go...